test/metabase/public_settings/premium_features_test.clj

```diff
@@ -4,6 +4,7 @@
             [clojure.test :refer :all]
             [metabase.models.user :refer [User]]
             [metabase.public-settings.premium-features :as premium-features]
+            [metabase.test.util :as tu] ;; Circular reference with mt in here, so use tu
```

Comment:
You got a circular reference when using `[metabase.test :as mt]`? I just tried it locally and didn't see that. Curious how that was happening.

Reply:
Did you run the tests from the command line or the repl?

Reply:
From the command line

Reply:
I just verified, there doesn't seem to be a circular dep issue. Not sure what I was seeing yesterday. I'll fixup now. Thanks @noahmoss

Reply:
That's right! lsp was just a bit too helpful.